UnityProject/Assets/Scripts/Core/Chat/Chat.cs

```diff
@@ -191,6 +191,43 @@ public static void AddChatMsgToChat(PlayerInfo sentByPlayer, string message, Cha
 		InvokeChatEvent(chatEvent);
 	}
 
+	/// <summary>
+	/// ServerSide Only, note there is no validation of message contents here for this type, normal player messages do no go this route
+	/// Chat modifiers do not work here
+	/// </summary>
+	public static void AddChatMsgToChat(string message, ChatChannel channels, Loudness loudness = Loudness.NORMAL)
```

Comment:
if it is serverside only and it can't have the ``Server`` attribute from networked behaviours, maybe add ``Server`` to the name of the method.